loki/README.md

```diff
@@ -136,7 +151,7 @@ When the status of the applied LogPipeline resource turns into `Running`, the un
 </div>
 
 >**NOTE:**
->- The used output plugin configuration uses all labels of a Pod to label the Loki log streams. Performance-wise, such segregation of the log streams might be not optimal. Follow [Loki's labelling best practices](https://grafana.com/docs/loki/latest/best-practices/) for a tailor-made setup that fits your workload configuration.
+>- The used output plugin configuration uses a static label map to assign labels of a Pod to Loki log streams. Activating the `     auto_kubernetes_labels` feature for using all labels of a Pod is not recommended performance-wise. Follow [Loki's labelling best practices](https://grafana.com/docs/loki/latest/best-practices/) for a tailor-made setup that fits your workload configuration.
```

Comment:
```suggestion
>- The used output plugin configuration uses a static label map to assign labels of a Pod to Loki log streams. It's not recommended to activate the `     auto_kubernetes_labels` feature for using all labels of a Pod because it affects the performance. Follow [Loki's labelling best practices](https://grafana.com/docs/loki/latest/best-practices/) for a tailor-made setup that fits your workload configuration.
```